src/ocm/components/clusterWizard/wizardTransition.ts

```diff
@@ -37,18 +46,48 @@ export const getClusterWizardFirstStep = (
     }
     return 'static-ip-network-wide-configurations';
   }
+  const step = getStepIdByHostValidations(hosts);
```

Comment:
This function should be moved down to line 56 because it's not fast to compute and no need to do it if it's not going to be used.

Also the name could hint better that it's the first step in which there are failing validations.
`getStepForFailingHostValidations` or something similar